packages/babel-plugin-transform-decorators/README.md

```diff
@@ -46,24 +46,42 @@ function enumerable(value) {
 }
 ```
 
-## Installation
+## Installation & Usage
```

Comment:
Nit: to keep it consistent with the rest of the docs let's keep Installation and Usage as separate sections